pokemon_battle_arena/pokemon_battle_arena.sublime-workspace

```diff
@@ -9,39 +9,7 @@
 			],
```

Comment:
The `sublime-workspace` file shouldn't be checked into git. Better remove it with `git rm pokemon_battle_arena.sublime-workspace` and add it to your `.gitignore` file.